corehq/apps/locations/tests/test_dbaccessors.py

```diff
@@ -100,3 +124,33 @@ def test_generate_user_ids_from_primary_location_ids_from_couch(self):
             ),
             [self.varys._id, self.tyrion._id, self.daenerys._id]
         )
+
+    # def test_generate_user_ids_from_primary_location_ids_es(self):
+    #     self.george.set_location(self.domain, self.pentos)
+    #     self.assertItemsEqual(
+    #         get_user_ids_from_primary_location_ids(
+    #             self.domain, [self.pentos.location_id]
+    #         ).keys(),
+    #         [self.varys._id]
+    #     )
+    #     self.george.unset_location(self.domain)
+
+    def test_get_user_ids_from_assigned_location_ids(self):
+        self.assertItemsEqual(
+            get_user_ids_from_assigned_location_ids(
+                self.domain, [self.meereen.location_id]
+            ).keys(),
+            [self.tyrion._id, self.daenerys._id]
+        )
+
+    def test_get_users_location_ids(self):
+        self.assertItemsEqual(
+            get_users_location_ids(self.domain, [self.varys._id, self.tyrion._id]),
+            [self.meereen._id, self.pentos._id]
+        )
+
+    def test_user_ids_at_locations(self):
+        self.assertItemsEqual(
+            user_ids_at_locations([self.meereen._id]),
```

Comment:
@esoergel I think this method never returned webusers because [user mapping](https://github.com/dimagi/commcare-hq/blob/master/corehq/pillows/mappings/user_mapping.py#L56) doesn't have an `domain_memberships.assigned_location_ids` property, so that's a bug that probably affected no one. Since it doesn't seem to have ever got reported, I am leaning towards just adding a disclaimer under `user_ids_at_locations` and updating `UserES.location_id` filter likewise, instead of fixing it as that would require a reindex. What do you think?

Reply:
yikes, I didn't know about that.  Fixing does seem out of scope, but I like Farid's idea of renaming to be explicit about what it does.

Reply:
Yes, good idea, updated.

Reply:
However, the users index is pretty small, so reindexing it shouldn't take too long if you did want to add this...